### PR TITLE
TL translation for environment.asc

### PR DESCRIPTION
**Pull request by Cyyy1998**

### CODE REVIEW

**Comment by severinolorillajr** [comment]
@Cyyy1998 , thank you for your contribution.
May I know who referred you to this project?
We already have a team working on this project.
But we can accept this contribution of course. But before that..

We also have few guidelines for contributions on this project.
One of which is to review other people's work in order to have your PR merged.
This is to ensure that every contributor is helping ensure the quality of contributions.
Thanks.

**Comment by Cyyy1998** [comment]
Sir jnavila referred me he said try to contribute to this project and wait if you approved it

**Comment by Cyyy1998** [comment]
I want to  be part your team if you give me chance

**Comment by severinolorillajr** [comment]
I don't know anyone named jnavila on Utopian or here at GitHub.
But in any case, you can join us at https://discord.gg/ujJzQB.
I will have a few projects there soon which are open to all Filipino translators.
This project is just a test project to formalize the process (work in progress), which I gave to my peers at work to test.

I will accept this contribution, but please also review the pending PR's we have here.
Then let me know once done so we can review and merge your work as well.

I will just update you on our Discord server, once the new project opens.

**Comment by Cyyy1998** [comment]
So its mean you will merged this project?

**Comment by severinolorillajr** [comment]
@Cyyy1998 , yes I just need you to review the other PR.
I have gone over your translation, so far so good.
But in order to have your work merged, review to be reviewed.
This will be our usual process if you want to be part of the team.

And after we have merged this, just wait for my signal regarding the new project.
I will update you once the new project is open.

https://github.com/progit2-tl/progit2/pull/13
This is a sample review process.

**Comment by Cyyy1998** [comment]
Okay thanks God bless

File: book/10-git-internals/sections/environment.asc [hunk 1]
```diff
@@ -1,137 +1,184 @@
 === Environment Variables
-
+===Mga Variable sa Kapaligiran
 Git always runs inside a `bash` shell, and uses a number of shell environment variables to determine how it behaves.
 Occasionally, it comes in handy to know what these are, and how they can be used to make Git behave the way you want it to.
 This isn't an exhaustive list of all the environment variables Git pays attention to, but we'll cover the most useful.
+Laging tumatakbo ang Git sa loob ng isang bash shell, at gumagamit ng isang bilang ng mga variable ng kapaligiran ng shell upang matukoy kung paano ito kumikilos. Paminsan-minsan, ito ay madaling gamitin upang malaman kung ano ang mga ito, at kung paano sila maaaring magamit upang gawing kumilos ang Git sa paraang gusto mo. Ito ay hindi isang malawakan na listahan ng lahat ng mga variable ng kapaligiran Git ay binabayaran ng pansin, ngunit tutugunan namin ang pinakamahalaga.
```
Comment (severinolorillajr) [inline book/10-git-internals/sections/environment.asc]: `bash shell`

File: book/10-git-internals/sections/environment.asc [hunk 2]
```diff
@@ -1,137 +1,184 @@
...
 
 
 ==== Global Behavior
+==== Ugali ng Global
 
 Some of Git's general behavior as a computer program depends on environment variables.
+Ang ilan sa pangkalahatang pag-uugali ng Git bilang isang programa sa computer ay depende sa mga variable ng kapaligiran.
```
Comment (severinolorillajr) [inline book/10-git-internals/sections/environment.asc]: *nakadepende

**Comment by severinolorillajr** [comment]
@Cyyy1998 Please check all instances of texts like this  *`GIT_INDEX_FILE`*
Don't remove the ` and * as this is needed for formatting the text.

Also, I hope you can spend some time as well to review other pending PRs. Thanks.

**Review by Cyyy1998 — COMMENTED** [review]
Its now edited

**Comment by severinolorillajr** [comment]
Check all instances of texts inside `___`.
You have removed the asterisks outside *`___`*

**Review by Cyyy1998 — COMMENTED** [review]
Done adding

**Review by Cyyy1998 — COMMENTED** [review]
Final

**Comment by severinolorillajr** [comment]
One last thing..
You forked a wrong repository.
You should use this.
https://github.com/progit2-tl/progit2
please close this PR and create a new one.

And again please also review the other pending PR's.

**Comment by Cyyy1998** [comment]
Ok thanks

**Comment by Cyyy1998** [comment]
I forked progit-tl

**Comment by severinolorillajr** [comment]
Checked *'HOME'*  this is still wrong.. use *`HOME`*.
Hope you can see the difference this is not a single quote but a grave accent.
Also you did not delete the english texts of the paragraphs that you have translated.

**Comment by severinolorillajr** [comment]
Checked \'HOME\' this is still wrong.. use  \`HOME\`   (without the spaces)
Hope you can see the difference this is not a single quote but a grave accent.

**Comment by Cyyy1998** [comment]
Please check my last review. Thank you

**Review by Cyyy1998 — COMMENTED** [review]
done

**Pull request merged**